lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml

```diff
@@ -60,6 +76,17 @@ silence_alert_from_detail:
   action: input_silence_comments
   action: perform_silence
   action: check_silence_detail
+silence_alert_from_detail_check_fields:
+  params:
+    comment_text: silence comments is a must from ocp 4.50
+    duration: 6h
+  action: click_silence_alert
+  action: set_silence_duration
+  action: uncheck_start_from_now
+  action: check_regular_expression_link
```

Comment:
how about checked_regular_expression

Reply:
I suppose check is better, here what I want to do is check is the link exists

Reply:
ok